python/lsst/dax/obscore_generator/gen_img_obscore.py

```diff
@@ -20,20 +20,19 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import math
 import os
-import uuid
 import urllib.parse as UrlParser
-import math
-import pandas as pd
-import json
+import uuid
 
-# import cProfile
 import click
```

Comment:
Is this old code that should all be deleted? I'm wondering about the use of `click` here and in `combine_obscore_csv.py`.